util/query.py

```diff
@@ -446,27 +447,34 @@ def _queryVolsOwners(
         result = submitQuery(query, chainID)
         if "errors" in result:
             raise AssertionError(result)
-        new_orders = result["data"]["orders"]
+        new_orders = result["data"]["fixedRateExchangeSwaps"]
```

Comment:
In DF, we want to account for 
1. selling of the data asset
2. consuming of the token
3. And ideally: include free assets where price = gas cost)

Before the PR, it did all of (1)(2)(3) albeit with a potential error. 

What this PR does: switch "orders" (the first step of a _consume_ - 2) with "fixedRateExchangeSwap" (a sale - 1).

What's the coverage of this PR?
- has (1)
- doesn't have (2)
- doesn't have (3). 

If needed we could skip (3) temporarily, I guess. 

But should we be skipping (2) ? I'm dubious.